frontend/src/features/leases/detail/LeasePages/payment/table/PaymentsForm.tsx

```diff
@@ -2,16 +2,19 @@ import { Button } from 'components/common/form';
 import { TableProps } from 'components/Table/Table';
 import Claims from 'constants/claims';
 import { useFormikContext } from 'formik';
+import useDeepCompareMemo from 'hooks/useDeepCompareMemo';
 import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
 import { IFormLease } from 'interfaces';
 import { IFormLeaseTerm } from 'interfaces/ILeaseTerm';
-import { orderBy } from 'lodash';
+import _, { noop, orderBy } from 'lodash';
```

Comment:
importing only the methods we want from lodash helps with tree-shaking. Would you consider importing `find` directly here? It seems the only usage of `_` here...